src/main/java/mate/model/Car.java

```diff
@@ -9,6 +9,16 @@ public class Car {
     private Manufacturer manufacturer;
     private List<Driver> drivers;
 
+    public Car() {
+
+    }
+
+    public Car(String model, Manufacturer manufacturer, List<Driver> drivers) {
+        this.model = model;
+        this.manufacturer = manufacturer;
+        this.drivers = drivers;
+    }
```

Comment:
```suggestion
    public Car(String model, Manufacturer manufacturer) {
        this.model = model;
        this.manufacturer = manufacturer;
        drivers = new ArrayList();
    }
```

я б так написала, що б не передавати порожній список

Reply:
@MaliukDaria зрозумів, дякую за пораду.